Brokerages/GDAX/GDAXBrokerage.cs

```diff
@@ -99,7 +93,13 @@ public override bool PlaceOrder(Orders.Order order)
 
                 if (order.Type == OrderType.Market)
                 {
-                    OnOrderEvent(new OrderEvent(order, DateTime.UtcNow, (decimal)raw.fill_fees, "GDAX Order Event") { Status = OrderStatus.Filled });
+                    Logging.Log.Trace("GDAXBrokerage.PlaceOrder(Market): Response: " + response.Content);
+                    OnOrderEvent(new OrderEvent(order, DateTime.UtcNow, (decimal) raw.FillFees, "GDAX Order Event")
+                    {
+                        Status = OrderStatus.Filled,
```

Comment:
think this should be:
                        FillPrice = raw.Price,
                        FillQuantity = raw.FilledSize
This change should get all unit tests passing.

Reply:
Mono 5.+ web sockets worked well but caused a memory leak. We've revered to Mono 4.6.1.5 until can patch the memory leak or write a custom websocket implementation from scratch.